src/SUMMARY.md

```diff
@@ -220,6 +223,22 @@
 * [Third-party resources](tutorials/third-party.md)
 * [Changing regions](tutorials/region-migration.md)
 
+## Dedicated
+
+* [Overview](dedicated/overview.md)
```

Comment:
Also, you're missing 

```
* [Onboarding](dedicated/overview/onboarding.md)
```

Reply:
They're all blank pages because their equivalent doesn't exist on the old ent-docs site.  I was just trying to maintain structure for review purposes.

So... I guess we can't merge this yet anyway. :smile:

Reply:
If you're going to maintain structure, then add a link to the new page. That's sensible anyway since people may have bookmarked things.

Reply:
It's a totally different domain so I don't expect bookmarks to work anyway.  My current thinking is to replace ent.docs.platform.sh with a trivial redirect to just `docs.platform.sh` and be done with it.  A lot of pages are now absent or heavily refactored and merged in elsewhere.  So "if you're going to maintain structure" isn't really true.